anjani/error.py

```diff
@@ -135,3 +132,7 @@ def __init__(self, old_plugin: Type["Plugin"], new_plugin: Type["Plugin"]) -> No
         self.old_plugin = old_plugin
         self.new_plugin = new_plugin
         super().__init__(f"Plugin '{old_plugin.name}' ({old_plugin.__name__}) already exists")
+
+
+class StopPropagation(AnjaniException):
```

Comment:
Don't think that `StopPropagation` class should be in the scope of error also inheriting `AnjaniException`